build.go

```diff
@@ -183,7 +183,10 @@ func Build(f BuildFunc, options ...Option) {
 		return
 	}
 
-	cnbPath := filepath.Clean(strings.TrimSuffix(config.args[0], filepath.Join("bin", "build")))
+	cnbPath := os.Getenv("CNB_BUILDPACK_DIR")
+	if cnbPath == "" {
+		cnbPath = filepath.Clean(strings.TrimSuffix(config.args[0], filepath.Join("bin", "build")))
+	}
```

Comment:
I think its a bit clearer to use `os.LookupEnv` here as it provides an API that tells you if the env var is not set, which is what we really care about, instead of switching on the contents of the env var. While it should never really be an empty string, I think using `os.LookupEnv` is a bit more intention revealing.

```suggestion
	cnbPath, ok := os.LookupEnv("CNB_BUILDPACK_DIR")
	if !ok {
		cnbPath = filepath.Clean(strings.TrimSuffix(config.args[0], filepath.Join("bin", "build")))
	}
```